### PR TITLE
Update Spanish Translation

### PR DESCRIPTION
**Pull request by diegombeltran**
* Fix previous suggestions
* Adjust changes to PR limit
* Use UTF-8

#7186

### CODE REVIEW

**Comment by diegombeltran** [comment]
Hi again,

I'm very sorry. I messed while rebasing, so I started over again. Please excuse my clumsiness.
The PR should be 400 exact changes. I removed the problematic lines (Button sizes) until a solution comes up.

File: files/lang/es.po [hunk 1]
```diff
@@ -1366,10 +1359,10 @@ msgid ""
 msgstr ""
 
 msgid "Blood is Thicker"
-msgstr ""
+msgstr "La Sangre es Espesa"
```
[the remark this answers is not in the document]
Comment (Joarico) [reply files/lang/es.po:1362]: @zenseii will the game support accents in spanish and symbols like "ñ"?

[the remark this answers is not in the document]
Comment (Joarico) [reply files/lang/es.po:1362]: > > @zenseii will the game support accents in spanish and symbols like "ñ"?
> 
> @Joarico, yes it already supports these accents. The only instance these accents aren't fully supported yet are for the buttons; those that are all capital letters.
> 
> "La Sangre es más Espesa" does not fit indeed: image

Then it's OK. I have already reviewed the whole PR and didn't found any issue in it.

File: files/lang/es.po [hunk 2]
```diff
@@ -2816,93 +2799,94 @@ msgid "Resource cost:"
...
 
 msgid "Need: "
-msgstr ""
+msgstr "Necesita: "
 
 msgid "Load Game"
-msgstr "Cargar juego"
+msgstr ""
+"Cargar\n"
+"Partida"
```
[the remark this answers is not in the document]
Comment (diegombeltran) [reply files/lang/es.po:2810]: It's not needed. I fixed it in the new commit.

File: files/lang/es.po [hunk 3]
```diff
@@ -2816,93 +2799,94 @@ msgid "Resource cost:"
...
-#, fuzzy
 msgid "No save files to load."
-msgstr "Sin hechizos que lanzar."
+msgstr "No hay partidas para cargar."
 
 msgid "New Game"
-msgstr "Nueva partida"
+msgstr ""
+"Nueva\n"
+"Partida"
```
[the remark this answers is not in the document]
Comment (diegombeltran) [reply files/lang/es.po:2818]: Fixed

**Comment by diegombeltran** [comment]
> Well done, @diegombeltran!
> 
> This PR is ready whenever you are, @ihhub :)

Nice!

I will continue translating the strings. Should I wait until this PR get merged?

Thank you for your time reviewing this @Joarico @zenseii @ihhub

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@diegombeltran , thank you so much for your contribution to the project!

**Pull request merged**